ecosystem/nft-metadata-crawler-parser/migrations/2023-06-27-001532_create_tables/up.sql

```diff
@@ -13,6 +13,8 @@ CREATE TABLE IF NOT EXISTS nft_metadata_crawler.parsed_token_uris (
   inserted_at TIMESTAMP NOT NULL DEFAULT NOW()
 );
 
+CREATE TABLE nft_metadata_crawler.ledger_infos (chain_id BIGINT UNIQUE PRIMARY KEY NOT NULL);
```

Comment:
Create TABLE if not exists. So that you don't get into trouble for duplicating the migration.